app/views/profiles/_show.html.erb

```diff
@@ -68,7 +68,7 @@
     <% if partner_profile_fields.include?('population_served') || partner_profile_fields.empty? %>
       <h2 class='text-2xl underline'>Population Served</h2>
       <p>Income Requirement
-      Description: <%= partner_profile.income_requirement_desc %></p>
+      Description(s): <%= partner_profile.income_requirement_desc %></p>
```

Comment:
@cielf Does this one need to be pluralized as well or shall we keep the original version of the label?

Reply:
Original please.